client/src/index.css

```diff
@@ -1,4 +1,25 @@
+html,
 body {
+  height: 100%;
+  width: 100%;
+}
+
+#root {
+  height: 100%;
+  width: 100%;
+  display: flex;
+}
```

Comment:
I wouldn't do this, as `#root` is a React-specific element, so this app wouldn't work if you would port it to another framework.

Reply:
Remove this styles and change `.App` so it behaves as expected.